script.js

```diff
@@ -0,0 +1,100 @@
+"use strict";
+// form issue 해결
+// submit이벤트가 발생하면 브라우저가refresh된다.
+// e.preventDefault()를 통해서 브라우저의 refresh를 막고
+// 새로운 변수 location에 input값 저장해서 textContent로 출력
+const Form = document.querySelector("form");
```

Comment:
```suggestion
const todoInputFormElement = document.querySelector("form");
```

자바스크립트에서 변수 이름은 일반적으로 소문자로 짓습니다!